.travis.yml

```diff
@@ -20,7 +20,11 @@ jobs:
         - cd backend
       install:
         - npm install
+        - cd node_modules/postman-templating
+        - npm install
+        - npm run test
       script:
+        - cd ../../
```

Comment:
I don't really like this portion; makes our deployment process harder to understand. I have to run `npm install` on `node_modules/postman-templating` because it seems like npm install creates a sym link to the folder for local modules. Perhaps I can `npm install` on the folder before I run `npm install` on backend and worker.